roles/compliance/tasks/create_scap_report.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# This could be a loop over a map of scans. In fact, has to be include_tasks
+# Also extract the results and create the report.
+
+- name: Set base name
+  ansible.builtin.set_fact:
+    base_name: "{{ bzip_file.path[:-10] }}" # we expect files to end in ".xml.bzip2"
+
+- name: Unzip the file
+  ansible.builtin.shell: "bunzip2 -c {{ bzip_file.path }} > {{ base_name }}.xml"
```

Comment:
What about using https://docs.ansible.com/ansible/latest/collections/ansible/builtin/unarchive_module.html ?

Reply:
Unfortunately  not.  According to the doc, the unarchive module does not handle .gz files, .bz2 files, .xz, or .zst files that do not contain a .tar archive.

Reply:
aw dang. good catch.